docs/source/search/_intro.rst

```diff
@@ -0,0 +1,14 @@
+Indico’s version 3.0 introduced a brand new reusable and agnostic search module
+backed up by your SQL storage by default. This module can however be decomposed
+into multiple providers, supporting any external service through a plugin.
```

Comment:
you can't have more than one at the same time; not sure if "multiple providers" isn't misleading